visualizer/loading-spinner.css

```diff
@@ -0,0 +1,58 @@
+body:not(.initialized):before,
+body:not(.initialized):after {
+  width: 124px;
+  height: 120px;
+  position: absolute;
+  left: 50%;
+  top: 50%;
+  margin-top: -60px;
+  margin-left: -60px;
+  z-index: 9;
+}
+
+body:not(.initialized) {
+  position: relative;
+  z-index: 3;
+}
+
+body:not(.initialized):before{
+  animation: loading 1.6s infinite linear;
+  border-bottom: 4px solid var(--flame-orange);
+  border-left: 1px solid var(--primary-grey);
+  border-radius: 50%;
+  border-right: 1px solid var(--primary-grey);
+  border-top: 4px solid var(--flame-orange);
+  content: "";
+  display: block;
+  transform: rotate(0deg);
+}
+
+body:not(.initialized):after{
+  align-items: center;
+  color: var(--flame-orange);
+  display: flex;
+  font-variant: all-small-caps;
+  font-weight: bold;
+  justify-content: center;
+  position: absolute;
+  text-align: center;
+  color: white;
```

Comment:
Can we use `var(--max-contrast)` so it's one less thing to change when we add light theme (which will probably have a CLI flag like presentation mode)